### PR TITLE
[fix] Highplot axis ticks overlapping

### PR DESCRIPTION
**Pull request by KaroMourad**
- add a limit to Highplot axis ticks count
- tune charts minimum ticks count and minimum dimensions

### CODE REVIEW

**Comment by KaroMourad** [comment]
Resolves #1594

**Review by VkoHov — APPROVED** [review]
lgtm!

File: aim/web/ui/src/components/ScatterPlot/styles.scss [hunk 1]
```diff
@@ -7,8 +7,8 @@
   position: relative;
   overflow: hidden;
   user-select: none;
-  min-width: 100px;
-  min-height: 100px;
+  min-width: 150px;
```
Comment (rubenaprikyan) [inline aim/web/ui/src/components/ScatterPlot/styles.scss:10]: Maybe use `rem` here?

Comment (KaroMourad) [reply aim/web/ui/src/components/ScatterPlot/styles.scss:10]: yeah , will replace it

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Pull request merged**